src/main.cc

```diff
@@ -997,6 +997,7 @@ mainReconfigureFinish(void *)
     reconfiguring = 0;
 }
 
+/// rotates the logs of the worker and other kid processes; \sa masterRotate()
 static void
 mainRotate(void)
```

Comment:
Single-process instances (e.g., `squid -N`) use this old mainRotate() function, not the new masterRotate(). In single-process instances, that only process is a _worker_, so the proposed description works OK for them as well. If you can suggest a better wording (that would cover all SMP and non-SMP cases), please do!